bin/ch-builder2tar

```diff
@@ -128,6 +128,32 @@ EOF
     tar rf "$tar" -b1 -P --xform="s|${temp}|ch/environment|" "$temp"
     rm "$temp"
 
+    echo "adding metadata"
+    temp=$(mktemp --tmpdir ch-builder2tar.XXXXXX)
+    buildah inspect --format='{{print "{\n"}}'\
+'{{printf "  \"arch\": %q,\n" .Docker.Architecture}}'\
```

Comment:
Why is this `printf` instead of `print` like the others?

Reply:
I used `printf` when printing a variable to get the formatting right and `print` otherwise.

Reply:
`should we bother with supporting Buildah since it's experimental?` That's a good question. I mostly added it because they worked quite similarly in that buildah replicates the metada docker provides, but we could remove the buildah one for just docker. What would like to do?